### PR TITLE
Avoid log error on 404

### PR DESCRIPTION
**Pull request by allevo**

### CODE REVIEW

File: test/404s.test.js [hunk 1]
```diff
@@ -443,3 +443,47 @@ test('setNotFoundHandler should not suppress duplicated routes checking', t => {
     t.ok(err)
   })
 })
+
+test('log debug for 404', t => {
+  t.plan(1)
+
+  const Writable = require('stream').Writable
+  class LogWritable extends Writable {
```
Comment (jsumners) [inline test/404s.test.js]: Can we please lose the `class` syntax and just pass an object literal with a `_write` property?

**Comment by mcollina** [comment]
I think they should be logged as info.

**Comment by jsumners** [comment]
Yeah, `log.error` for `5xx` codes and `info` or `warn` for `4xx` makes more sense. I lean toward `warn`. I really don't care about `404`s, particularly since I'm in an environment where we senselessly run Nessus against everything.

**Comment by allevo** [comment]
5xx => error
4xx => info

**Pull request merged**